app/src/main/java/com/amaze/filemanager/fragments/FtpServerFragment.java

```diff
@@ -82,6 +82,8 @@
     private MainActivity mainActivity;
 
     private TextView statusText, url, username, password, port, sharedPath;
+    public static final String TAG = "FTPServerFragment";
```

Comment:
Perhaps `FtpServerFragment.class.getSimpleName()` is better?

Reply:
The code in MainActivity#2044 doesn't seem to support dynamic method call :P This will be required anyways.